api/api.go

```diff
@@ -128,6 +128,31 @@ func Submit(url string, iter *Iteration) (*Submission, error) {
 	return ps.Submission, nil
 }
 
+// List available problems for a language
+func List(language, host string) ([]string, error) {
```

Comment:
Yeah, agreed. Let's let the api worry about urls (functions or hard-coded), and the function arguments can be whatever information is needed.